src/Alert/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cc from "classcat";
+import Row from "../Row";
+
+const noop = () => {};
+
+/**
+ * Inline system message, for a sepcific region of a page.
+ * The `isActive` prop is used to hide and show the Alert to ensure the Alert
+ * is always rendered in an [ARIA live region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions)
+ * for accessibility.
+ */
+const Alert = ({
+  isActive = false,
+  isDismissable = true,
+  onUserDismiss = noop,
+  kind = "info",
+  children,
+}) => {
+  const iconName = kind === "success" ? "check" : "info";
+
+  return (
+    <div aria-live="polite">
```

Comment:
This is the live region. When you use an `Alert`, this div will _always_ get rendered. This is useful because assistive technology will announce the alert that renders inside it when `isActive` changes to `true`.